From: Peter Lemenkov <lemenkov@gmail.com>
Date: Tue, 31 Oct 2023 19:41:13 +0100
Subject: [PATCH] Fedora-specific: installation directory on 64-bit arches

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a9fb00f7e..b067a40ab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,7 +83,7 @@ list(APPEND CMAKE_PREFIX_PATH "${PROJECT_SOURCE_DIR}/deps/install")
 #-----------------------------------------------------------------------------#
 
 set(INCLUDE_INSTALL_DIR include)
-set(LIBRARY_INSTALL_DIR lib)
+set(LIBRARY_INSTALL_DIR lib${LIB_SUFFIX})
 set(RUNTIME_INSTALL_DIR bin)
 
 #-----------------------------------------------------------------------------#
