From: Jerry James <loganjerry@gmail.com>
Date: Sun, 26 Apr 2020 17:26:14 -0600
Subject: [PATCH] Add patch to adapt to changes in 5.7.0


diff --git a/src/prop/cryptominisat.cpp b/src/prop/cryptominisat.cpp
index cf23758f1..97e7a350b 100644
--- a/src/prop/cryptominisat.cpp
+++ b/src/prop/cryptominisat.cpp
@@ -73,15 +73,8 @@ CryptoMinisatSolver::CryptoMinisatSolver(StatisticsRegistry* registry,
 
 void CryptoMinisatSolver::init()
 {
-  d_true = newVar();
-  d_false = newVar();
-
-  std::vector<CMSat::Lit> clause(1);
-  clause[0] = CMSat::Lit(d_true, false);
-  d_solver->add_clause(clause);
-
-  clause[0] = CMSat::Lit(d_false, true);
-  d_solver->add_clause(clause);
+  d_true = undefSatVariable;
+  d_false = undefSatVariable;
 }
 
 CryptoMinisatSolver::~CryptoMinisatSolver() {}
@@ -158,10 +151,32 @@ SatVariable  CryptoMinisatSolver::newVar(bool isTheoryAtom, bool preRegister, bo
 }
 
 SatVariable CryptoMinisatSolver::trueVar() {
+  if (d_true == undefSatVariable) {
+    d_true = newVar();
+    d_false = newVar();
+
+    std::vector<CMSat::Lit> clause(1);
+    clause[0] = CMSat::Lit(d_true, false);
+    d_solver->add_clause(clause);
+
+    clause[0] = CMSat::Lit(d_false, true);
+    d_solver->add_clause(clause);
+  }
   return d_true;
 }
 
 SatVariable CryptoMinisatSolver::falseVar() {
+  if (d_true == undefSatVariable) {
+    d_true = newVar();
+    d_false = newVar();
+
+    std::vector<CMSat::Lit> clause(1);
+    clause[0] = CMSat::Lit(d_true, false);
+    d_solver->add_clause(clause);
+
+    clause[0] = CMSat::Lit(d_false, true);
+    d_solver->add_clause(clause);
+  }
   return d_false;
 }
 
