From: Jerry James <loganjerry@gmail.com>
Date: Wed, 20 Jan 2021 21:58:43 -0700
Subject: [PATCH] Fix FTBFS with recent LFSC versions


diff --git a/proofs/signatures/th_bv.plf b/proofs/signatures/th_bv.plf
index 934951a86..f0ced51c7 100644
--- a/proofs/signatures/th_bv.plf
+++ b/proofs/signatures/th_bv.plf
@@ -72,7 +72,6 @@
 
 (declare bvand bvop2)
 (declare bvor bvop2)
-(declare bvor bvop2)
 (declare bvxor bvop2)
 (declare bvnand bvop2)
 (declare bvnor bvop2)
@@ -88,7 +87,6 @@
 (declare bvshl bvop2)
 (declare bvlshr bvop2)
 (declare bvashr bvop2)
-(declare concat bvop2)
 
 ; bit vector unary operators
 (define bvop1
