--- ./src/decision/options.h.orig	2014-07-13 11:48:44.263172259 -0600
+++ ./src/decision/options.h	2014-07-13 11:48:44.263172259 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file decision/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/expr/expr_manager_template.h.orig	2014-07-13 11:32:35.479585444 -0600
+++ ./src/expr/expr_manager_template.h	2014-07-13 11:32:35.479585444 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_manager_template.h
+/*! \file expr_manager.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/expr_template.h.orig	2014-07-13 11:32:35.479585444 -0600
+++ ./src/expr/expr_template.h	2014-07-13 11:32:35.479585444 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_template.h
+/*! \file expr.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/kind_template.h.orig	2014-07-13 11:32:35.479585444 -0600
+++ ./src/expr/kind_template.h	2014-07-13 11:32:35.479585444 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file kind_template.h
+/*! \file kind.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/options.h.orig	2014-07-13 11:48:13.930308852 -0600
+++ ./src/expr/options.h	2014-07-13 11:48:13.930308852 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file expr/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/main/options.h.orig	2014-07-13 11:48:47.275258010 -0600
+++ ./src/main/options.h	2014-07-13 11:48:47.275258010 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file main/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/base_options.h.orig	2014-07-13 11:48:12.830277541 -0600
+++ ./src/options/base_options.h	2014-07-13 11:48:12.830277541 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file options/base_options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/base_options_template.h.orig	2014-07-13 11:32:35.579588292 -0600
+++ ./src/options/base_options_template.h	2014-07-13 11:32:35.579588292 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/options.h.orig	2014-07-01 12:37:15.061881767 -0600
+++ ./src/options/options.h	2014-07-01 12:37:15.061881767 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file options.h
+/*! \file options/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/parser/options.h.orig	2014-07-13 11:48:48.427290778 -0600
+++ ./src/parser/options.h	2014-07-13 11:48:48.427290778 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file parser/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/printer/options.h.orig	2014-07-13 11:48:37.070967532 -0600
+++ ./src/printer/options.h	2014-07-13 11:48:37.070967532 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file printer/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/proof/options.h.orig	2014-07-13 11:48:36.346946926 -0600
+++ ./src/proof/options.h	2014-07-13 11:48:36.346946926 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file proof/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/prop/options.h.orig	2014-07-13 11:48:35.870933374 -0600
+++ ./src/prop/options.h	2014-07-13 11:48:35.870933374 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file prop/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/smt/options.h.orig	2014-07-13 11:48:42.683127279 -0600
+++ ./src/smt/options.h	2014-07-13 11:48:42.683127279 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file smt/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/arith/options.h.orig	2014-07-13 11:48:23.866591680 -0600
+++ ./src/theory/arith/options.h	2014-07-13 11:48:23.866591680 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/arith/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/arrays/options.h.orig	2014-07-13 11:48:26.966679925 -0600
+++ ./src/theory/arrays/options.h	2014-07-13 11:48:26.966679925 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/arrays/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/booleans/options.h.orig	2014-07-13 11:48:14.450323654 -0600
+++ ./src/theory/booleans/options.h	2014-07-13 11:48:14.450323654 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/booleans/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/builtin/options.h.orig	2014-07-13 11:48:18.334434213 -0600
+++ ./src/theory/builtin/options.h	2014-07-13 11:48:18.334434213 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/builtin/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/bv/options.h.orig	2014-07-13 11:48:17.270403926 -0600
+++ ./src/theory/bv/options.h	2014-07-13 11:48:17.270403926 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/bv/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/datatypes/options.h.orig	2014-07-13 11:48:17.930422713 -0600
+++ ./src/theory/datatypes/options.h	2014-07-13 11:48:17.930422713 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/datatypes/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/idl/options.h.orig	2014-07-13 11:48:49.099309909 -0600
+++ ./src/theory/idl/options.h	2014-07-13 11:48:49.099309909 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/idl/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/options.h.orig	2014-07-13 11:48:15.082341645 -0600
+++ ./src/theory/options.h	2014-07-13 11:48:15.082341645 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/quantifiers/options.h.orig	2014-07-13 11:48:32.886848435 -0600
+++ ./src/theory/quantifiers/options.h	2014-07-13 11:48:32.886848435 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/quantifiers/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/sets/options.h.orig	2014-07-13 11:48:49.839330972 -0600
+++ ./src/theory/sets/options.h	2014-07-13 11:48:49.839330972 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/sets/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/strings/options.h.orig	2014-07-13 11:48:34.382891021 -0600
+++ ./src/theory/strings/options.h	2014-07-13 11:48:34.382891021 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/strings/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/uf/options.h.orig	2014-07-13 11:48:25.926650321 -0600
+++ ./src/theory/uf/options.h	2014-07-13 11:48:25.926650321 -0600
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/uf/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/util/tls.h.in.orig	2014-07-13 11:32:36.123603789 -0600
+++ ./src/util/tls.h.in	2014-07-13 11:32:36.123603789 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file tls.h.in
+/*! \file tls.h
  ** \verbatim
  ** Original author: ACSYS
  ** Major contributors: Morgan Deters
