--- config/abc.m4.orig	2018-06-25 15:13:17.531906649 -0600
+++ config/abc.m4	2018-07-05 14:55:03.244625126 -0600
@@ -35,21 +35,17 @@ elif test -n "$with_abc"; then
     AC_MSG_RESULT([no])
   fi
 
-  if ! test -d "$ABC_HOME" || ! test -x "$ABC_HOME/arch_flags"; then
+  if ! test -d "$ABC_HOME"; then
     AC_MSG_FAILURE([either $ABC_HOME is not an abc source tree or it's not yet built])
   fi
 
-  AC_MSG_CHECKING([for arch_flags to use with libabc])
-  libabc_arch_flags="$("$ABC_HOME/arch_flags")"
-  AC_MSG_RESULT([$libabc_arch_flags])
-  CVC4CPPFLAGS="${CVC4CPPFLAGS:+$CVC4CPPFLAGS }-I$ABC_HOME/src $libabc_arch_flags"
-  ABC_LDFLAGS="-L$ABC_HOME"
+  CVC4CPPFLAGS="${CVC4CPPFLAGS:+$CVC4CPPFLAGS }-I/usr/include/abc"
+  ABC_LDFLAGS=""
 
   dnl Try a bunch of combinations until something works :-/
   cvc4_save_LDFLAGS="$LDFLAGS"
   ABC_LIBS=
-  CPPFLAGS="$CPPFLAGS -I$ABC_HOME/src $libabc_arch_flags"
-  LDFLAGS="$LDFLAGS $ABC_LDFLAGS"
+  CPPFLAGS="$CPPFLAGS -I/usr/include/abc"
   AC_CHECK_HEADER([base/abc/abc.h], [], [AC_MSG_FAILURE([cannot find abc.h, the ABC header!])])
   AC_MSG_CHECKING([how to link abc])
   CVC4_TRY_ABC_WITH([])
--- config/antlr.m4.orig	2018-06-25 15:13:17.532906666 -0600
+++ config/antlr.m4	2018-07-05 14:56:06.052723809 -0600
@@ -35,6 +35,7 @@ been generated already. To obtain ANTLR
     case "$ANTLR_VERSION" in
       3.2|3.2.*) ANTLR_VERSION=3.2 ;;
       3.4|3.4.*) ANTLR_VERSION=3.4 ;;
+      3.5|3.5.*) ANTLR_VERSION=3.5 ;;
       *) AC_MSG_WARN([unknown version of antlr: $ANTLR_VERSION]);;
     esac
   fi
@@ -130,8 +131,8 @@ AC_DEFUN([AC_LIB_ANTLR],[
               }
             ])],
             [
-              AC_MSG_RESULT([found it (must be antlr3 3.4 or similar)])
-              if test -n "$ANTLR_VERSION" -a "$ANTLR_VERSION" != 3.4; then
+              AC_MSG_RESULT([found it (must be antlr3 3.5 or similar)])
+              if test -n "$ANTLR_VERSION" -a "$ANTLR_VERSION" != 3.5; then
                 AC_MSG_WARN([your antlr parser generator is version $ANTLR_VERSION, which doesn't match the library!])
               fi
             ],
--- config/boost.m4.orig	2018-06-25 15:13:17.539906789 -0600
+++ config/boost.m4	2018-07-05 14:56:32.388765183 -0600
@@ -71,7 +71,7 @@ dnl strip `\n' with backquotes, not the
 dnl boost_cv_lib_version='1_37\r' for instance, which breaks
 dnl everything else.
 dnl Cannot use 'dnl' after [$4] because a trailing dnl may break AC_CACHE_CHECK
-(eval "$ac_cpp conftest.$ac_ext") 2>&AS_MESSAGE_LOG_FD |
+(eval "$ac_cpp -P conftest.$ac_ext") 2>&AS_MESSAGE_LOG_FD |
   grep -v "^#" |
   tr -d '\r\n ' |
   $SED -n -e "$1" >conftest.i 2>&1],
--- config/cryptominisat.m4.orig	2018-06-25 15:13:17.542906841 -0600
+++ config/cryptominisat.m4	2018-07-05 14:57:42.402740101 -0600
@@ -36,7 +36,7 @@ elif test -n "$with_cryptominisat"; then
     AC_MSG_RESULT([no])
   fi
 
-  if ! test -d "$CRYPTOMINISAT_HOME" || ! test -x "$CRYPTOMINISAT_HOME/install/bin/cryptominisat5_simple" ; then
+  if ! test -d "$CRYPTOMINISAT_HOME" ; then
     AC_MSG_FAILURE([either $CRYPTOMINISAT_HOME is not an cryptominisat install tree or it's not yet built])
   fi
 
@@ -54,7 +54,7 @@ elif test -n "$with_cryptominisat"; then
     have_libcryptominisat=1
   fi
 
-  CRYPTOMINISAT_LDFLAGS="-L$CRYPTOMINISAT_HOME/install/lib"
+  CRYPTOMINISAT_LDFLAGS=""
 
 else
   AC_MSG_RESULT([no, user didn't request cryptominisat])
@@ -74,8 +74,8 @@ if test -z "$CRYPTOMINISAT_LIBS"; then
   cvc4_save_LDFLAGS="$LDFLAGS"
   cvc4_save_CPPFLAGS="$CPPFLAGS"
 
-  LDFLAGS="-L$CRYPTOMINISAT_HOME/install/lib"
-  CPPFLAGS="$CPPFLAGS -I$CRYPTOMINISAT_HOME/install/include"
+  LDFLAGS=""
+  CPPFLAGS="$CPPFLAGS -I/usr/include/cryptominisat5"
   LIBS="-lcryptominisat5 $1"
 
   AC_LINK_IFELSE(
--- configure.ac.orig	2018-06-25 15:33:12.372543152 -0600
+++ configure.ac	2018-07-05 14:58:49.284601972 -0600
@@ -294,7 +294,7 @@ if test $cross_compiling = "no"; then
   AC_MSG_CHECKING([whether C++ exceptions work])
   AC_LANG_PUSH([C++])
   AC_RUN_IFELSE(
-    AC_LANG_PROGRAM([#include <exception>], [[
+    [AC_LANG_PROGRAM([#include <exception>], [[
       int result = 1;
       try {
         throw std::exception();
@@ -302,7 +302,7 @@ if test $cross_compiling = "no"; then
         result = 0;
       }
       return result;
-    ]]),
+    ]])],
     [AC_MSG_RESULT([yes])],
     [AC_MSG_ERROR([C++ exceptions do not work.])]
   )
@@ -657,11 +657,6 @@ fi
 
 AC_MSG_RESULT([$enable_debug_symbols])
 
-if test "$enable_debug_symbols" = yes; then
-  CVC4CXXFLAGS="${CVC4CXXFLAGS:+$CVC4CXXFLAGS }-ggdb3"
-  CVC4CFLAGS="${CVC4CFLAGS:+$CVC4CFLAGS }-ggdb3"
-fi
-
 AC_MSG_CHECKING([whether to enable Valgrind instrumentation])
 
 AC_ARG_ENABLE([valgrind],
