--- config/symfpu.m4.orig	2018-06-25 15:13:17.555907069 -0600
+++ config/symfpu.m4	2018-07-07 10:34:08.415291211 -0600
@@ -28,10 +28,10 @@ elif test -n "$with_symfpu"; then
   if test -z "$SYMFPU_HOME" && test -e "$ac_abs_confdir/symfpu-CVC4/symfpu/core"; then
     SYMFPU_HOME="$ac_abs_confdir/symfpu-CVC4"
     AC_MSG_RESULT([yes, $SYMFPU_HOME])
-    have_symfpu_headers=1
   else
     AC_MSG_RESULT([no])
   fi
+  have_symfpu_headers=1
 else
   AC_MSG_RESULT([no, user didn't request symfpu])
   with_symfpu=no
