From: Jerry James <loganjerry@gmail.com>
Date: Tue, 11 Oct 2022 14:54:00 -0600
Subject: [PATCH] Drop python3-toml BR


diff --git a/src/options/CMakeLists.txt b/src/options/CMakeLists.txt
index 46bd17172..ecf4f7c03 100644
--- a/src/options/CMakeLists.txt
+++ b/src/options/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Check if the toml Python module is installed.
 execute_process(
   COMMAND
-  ${PYTHON_EXECUTABLE} -c "import toml"
+  ${PYTHON_EXECUTABLE} -c "import tomllib"
   RESULT_VARIABLE
     RET_TOML
   ERROR_QUIET
diff --git a/src/options/mkoptions.py b/src/options/mkoptions.py
index d9bc1a0bd..7d2f37849 100755
--- a/src/options/mkoptions.py
+++ b/src/options/mkoptions.py
@@ -46,7 +46,7 @@ import os
 import re
 import sys
 import textwrap
-import toml
+import tomllib
 
 ### Allowed attributes for module/option/alias
 
@@ -1309,7 +1309,8 @@ def mkoptions_main():
     # Parse files, check attributes and create module/option objects
     modules = []
     for filename in filenames:
-        module = parse_module(filename, toml.load(filename))
+        with open(filename, "rb") as f:
+            module = parse_module(filename, tomllib.load(f))
 
         # Check if long options are valid and unique.  First populate
         # g_long_cache with option.long and --no- alternatives if
