--- ./src/decision/options.h.orig	2013-12-06 16:07:43.586866550 -0700
+++ ./src/decision/options.h	2013-12-06 16:07:43.586866550 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file decision/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/expr/expr_manager_template.h.orig	2013-12-03 15:24:58.911994034 -0700
+++ ./src/expr/expr_manager_template.h	2013-12-03 15:24:58.911994034 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_manager_template.h
+/*! \file expr_manager.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/expr_template.h.orig	2013-11-27 09:42:59.916302990 -0700
+++ ./src/expr/expr_template.h	2013-11-27 09:42:59.916302990 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_template.h
+/*! \file expr.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/kind_template.h.orig	2013-11-11 10:21:58.180347624 -0700
+++ ./src/expr/kind_template.h	2013-11-11 10:21:58.180347624 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file kind_template.h
+/*! \file kind.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: Dejan Jovanovic
--- ./src/expr/options.h.orig	2013-12-06 16:07:22.010279788 -0700
+++ ./src/expr/options.h	2013-12-06 16:07:22.010279788 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file expr/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/main/options.h.orig	2013-12-06 16:07:45.234911367 -0700
+++ ./src/main/options.h	2013-12-06 16:07:45.234911367 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file main/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/base_options.h.orig	2013-12-06 16:07:21.118255532 -0700
+++ ./src/options/base_options.h	2013-12-06 16:07:21.118255532 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file options/base_options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/base_options_template.h.orig	2013-11-11 10:21:58.276350312 -0700
+++ ./src/options/base_options_template.h	2013-11-11 10:21:58.276350312 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/options/options.h.orig	2013-11-11 10:21:58.284350536 -0700
+++ ./src/options/options.h	2013-11-11 10:21:58.284350536 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file options.h
+/*! \file options/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/parser/options.h.orig	2013-12-06 16:07:46.070934102 -0700
+++ ./src/parser/options.h	2013-12-06 16:07:46.070934102 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file parser/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/printer/options.h.orig	2013-12-06 16:07:39.042742975 -0700
+++ ./src/printer/options.h	2013-12-06 16:07:39.042742975 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file printer/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/proof/options.h.orig	2013-12-06 16:07:38.522728834 -0700
+++ ./src/proof/options.h	2013-12-06 16:07:38.522728834 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file proof/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/prop/options.h.orig	2013-12-06 16:07:38.086716977 -0700
+++ ./src/prop/options.h	2013-12-06 16:07:38.086716977 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file prop/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/smt/options.h.orig	2013-12-06 16:07:42.546838267 -0700
+++ ./src/smt/options.h	2013-12-06 16:07:42.546838267 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file smt/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/arith/options.h.orig	2013-12-06 16:07:28.406453727 -0700
+++ ./src/theory/arith/options.h	2013-12-06 16:07:28.406453727 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/arith/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/arrays/options.h.orig	2013-12-06 16:07:31.002524327 -0700
+++ ./src/theory/arrays/options.h	2013-12-06 16:07:31.002524327 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/arrays/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/booleans/options.h.orig	2013-12-06 16:07:22.494292954 -0700
+++ ./src/theory/booleans/options.h	2013-12-06 16:07:22.494292954 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/booleans/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/builtin/options.h.orig	2013-12-06 16:07:25.114364203 -0700
+++ ./src/theory/builtin/options.h	2013-12-06 16:07:25.114364203 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/builtin/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/bv/options.h.orig	2013-12-06 16:07:24.114337006 -0700
+++ ./src/theory/bv/options.h	2013-12-06 16:07:24.114337006 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/bv/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/datatypes/options.h.orig	2013-12-06 16:07:24.714353323 -0700
+++ ./src/theory/datatypes/options.h	2013-12-06 16:07:24.714353323 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/datatypes/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/options.h.orig	2013-12-06 16:07:23.074308723 -0700
+++ ./src/theory/options.h	2013-12-06 16:07:23.074308723 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/quantifiers/options.h.orig	2013-12-06 16:07:35.278640613 -0700
+++ ./src/theory/quantifiers/options.h	2013-12-06 16:07:35.278640613 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/quantifiers/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/rewriterules/options.h.orig	2013-12-06 16:07:35.926658235 -0700
+++ ./src/theory/rewriterules/options.h	2013-12-06 16:07:35.926658235 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/rewriterules/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/theory/uf/options.h.orig	2013-12-06 16:07:30.142500939 -0700
+++ ./src/theory/uf/options.h	2013-12-06 16:07:30.142500939 -0700
@@ -28,7 +28,7 @@
 /* Edit the template file instead.                     */
 
 /*********************                                                        */
-/*! \file base_options_template.h
+/*! \file theory/uf/options.h
  ** \verbatim
  ** Original author: Morgan Deters
  ** Major contributors: none
--- ./src/util/tls.h.in.orig	2013-12-03 13:43:42.822811181 -0700
+++ ./src/util/tls.h.in	2013-12-03 13:43:42.822811181 -0700
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file tls.h.in
+/*! \file tls.h
  ** \verbatim
  ** Original author: ACSYS
  ** Major contributors: Morgan Deters
