--- src/expr/expr_manager_template.cpp.orig	2018-06-25 15:13:17.952914014 -0600
+++ src/expr/expr_manager_template.cpp	2018-07-05 14:39:39.136284939 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_manager_template.cpp
+/*! \file expr_manager.cpp
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Christopher L. Conway, Dejan Jovanovic
--- src/expr/expr_manager_template.h.orig	2018-06-25 15:13:17.953914032 -0600
+++ src/expr/expr_manager_template.h	2018-07-05 14:39:39.137284937 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_manager_template.h
+/*! \file expr_manager.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Dejan Jovanovic, Christopher L. Conway
--- src/expr/expr_template.h.orig	2018-06-25 15:13:17.958914119 -0600
+++ src/expr/expr_template.h	2018-07-05 14:39:39.138284935 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file expr_template.h
+/*! \file expr.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Dejan Jovanovic, Aina Niemetz
--- src/expr/kind_template.h.orig	2018-06-25 15:13:17.962914189 -0600
+++ src/expr/kind_template.h	2018-07-05 14:39:39.138284935 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file kind_template.h
+/*! \file kind.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Dejan Jovanovic, Andres Noetzli
--- src/expr/metakind_template.h.orig	2018-06-25 15:13:17.966914259 -0600
+++ src/expr/metakind_template.h	2018-07-05 14:39:39.139284933 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file metakind_template.h
+/*! \file metakind.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Andres Noetzli, Tim King
--- src/expr/type_checker_template.cpp.orig	2018-06-25 15:13:18.004914924 -0600
+++ src/expr/type_checker_template.cpp	2018-07-05 14:39:39.139284933 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file type_checker_template.cpp
+/*! \file type_checker.cpp
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Tim King, Andrew Reynolds
--- src/expr/type_properties_template.h.orig	2018-06-25 15:13:18.007914976 -0600
+++ src/expr/type_properties_template.h	2018-07-05 14:39:39.139284933 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file type_properties_template.h
+/*! \file type_properties.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Tim King
--- src/util/rational.h.in.orig	2018-06-25 15:13:19.197935796 -0600
+++ src/util/rational.h.in	2018-07-05 14:39:39.139284933 -0600
@@ -1,5 +1,5 @@
 /*********************                                                        */
-/*! \file rational.h.in
+/*! \file rational.h
  ** \verbatim
  ** Top contributors (to current version):
  **   Morgan Deters, Tim King
